command/transition.go

```diff
@@ -0,0 +1,77 @@
+package command
+
+import (
+	"encoding/json"
+	"github.com/ExpediaGroup/flyte-jira/client"
+	"github.com/HotelsDotCom/flyte-client/flyte"
+	"log"
+)
+
+var (
+	Transition = flyte.Command{
+		Name:         "Transition",
+		OutputEvents: []flyte.EventDef{transitionEventDef, transitionFailureEventDef},
+		Handler:      transitionHandler,
+	}
+
+	transitionEventDef = flyte.EventDef{
+		Name: "Transition",
+	}
+
+	transitionFailureEventDef = flyte.EventDef{
+		Name: "TransitionFailure",
+	}
+)
+
+type transitionRequest struct {
+	IssueId      string `json:"issueId"`
+	TransitionId string `json:"transitionId"`
+}
+
+type transitionPayload struct {
+	IssueId      string `json:"issueId"`
+	TransitionId string `json:"transitionId"`
+	RequestURL   string `json:"requestURL"`
```

Comment:
How this will be used?

Reply:
I'm returning the request URL with the payload https://github.com/ExpediaGroup/flyte-jira/blob/7a23c6872eb1cfb13fd11398be14f96614b750d1/command/transition.go#L62 - so, we can test the URL https://github.com/ExpediaGroup/flyte-jira/blob/7a23c6872eb1cfb13fd11398be14f96614b750d1/command/transition_test.go#L54 (it was one of your comments on the tests to test the URL).

Reply:
https://github.com/ExpediaGroup/flyte-jira/pull/23#discussion_r503809955